libs/openchallenges/ui/src/lib/navbar/navbar.component.html

```diff
@@ -1,30 +1,39 @@
-<!-- TODO: figure out if the <nav> should go inside of a <header> element. -->
-<nav class="sage-navbar" aria-label="Top Toolbar">
-  <a mat-button routerLink="/" aria-label="Home">
-    <img
-      class="logo"
-      src="/openchallenges-assets/images/openchallenges-color.svg"
-      alt="OpenChallenges"
-    />
-  </a>
+<nav aria-label="Navbar">
+  <mat-toolbar class="sage-navbar">
+    <a mat-button routerLink="/" aria-label="Home">
+      <img
+        class="logo"
+        src="/openchallenges-assets/images/openchallenges-color.svg"
+        alt="OpenChallenges"
+      />
+    </a>
+    <div fxFlex fxLayout fxLayoutAlign="end" fxHide.gt-sm>
+      <button mat-icon-button type="button" aria-label="Sidenav" (click)="onToggleSidenav()">
+        <mat-icon aria-hidden="true">menu</mat-icon>
+      </button>
+    </div>
+    <div fxFlex fxLayout fxLayoutAlign="start" fxHide.lt-md>
```

Comment:
`fxFlex` by default will take all available space if no value provided, which is the cause of nav item moving behavior @tschaffter pointed out. Probably most nav items needs some layout adjustments here.

I might need time to learn about `ngx-layout` for have a better understanding it. I suggest to save the `ngx-layout` changes to another pr, unless the usage of `ngx-layout` has already been showcased in the stand-up meeting.